pkg/sql/alter_type.go

```diff
@@ -55,13 +56,19 @@ func (n *alterTypeNode) startExec(params runParams) error {
 	var err error
 	switch t := n.n.Cmd.(type) {
 	case *tree.AlterTypeAddValue:
-		err = params.p.addEnumValue(params, n, t)
+		err = params.p.addEnumValue(params.ctx, n, t, tree.AsStringWithFQNames(n.n, params.Ann()))
 	case *tree.AlterTypeRenameValue:
-		err = params.p.renameTypeValue(params, n, t.OldVal, t.NewVal)
+		err = params.p.renameTypeValue(
+			params.ctx, n, t.OldVal, t.NewVal, tree.AsStringWithFQNames(n.n, params.Ann()),
```

Comment:
why did you change all of these to take in jobdescs? If it was cuz you don't have params.Ann(), you can get it from `p.ExtendedEvalContext().EvalCtx.Annotations`